test/cts/test/V1_3/depthwise_conv2d_quant8_signed.js

```diff
@@ -0,0 +1,236 @@
+// Generated file (from: depthwise_conv2d_quant8_signed.mod.py). Do not edit
+describe('CTS', function() {
+  const assert = chai.assert;
+  const nn = navigator.ml.getNeuralNetworkContext();
+
+  it('check result for Depthwise conv2d quant8 signed example-1', async function() {
+    // For 'Depthwise conv2d quant8 signed' example: examples_same
+    let model = await nn.createModel(options);
+    let operandIndex = 0;
+
+    let op15_value = [-124, -112, -124, -96, -124, -64, -124, 0];
+    let op45_expect = [-120, -80];
+
+    let type10 = {type: nn.TENSOR_QUANT8_ASYMM_SIGNED, dimensions: [1, 2, 2, 2], scale: 0.5, zeroPoint: -128};
```

Comment:
@cuiyanx Yes, here **zeroPoint** can be -128, referring to NNAPI [ANEURALNETWORKS_TENSOR_QUANT8_ASYMM_SIGNED](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaf06d1affd33f3bc698d0c04eceb23298a0f7c57dd01611105bbdd17922dacbba9)
```
zeroPoint: a 32 bit integer, in range [-128, 127].
```

so the comment for **zeroPoint** of dictionary [**OperandOptions**](https://github.com/intel/webml-polyfill/blob/master/docs/api.md#model) in [API doc](https://github.com/intel/webml-polyfill/blob/master/docs/api.md)
```javascript
  // zeroPoint: an integer, in range [0, 255]
  long zeroPoint;
```
needs to update, would you please submit a new pr for fixing it, thanks.

Reply:
@cuiyanx Ningxin has already modified this comment on PR #1156, so don't need to update it.